src/core/a-entity.js

```diff
@@ -539,7 +539,7 @@ var proto = Object.create(ANode.prototype, {
    */
   setAttribute: {
     value: function (attr, value, componentPropValue) {
-      var isDebugMode = this.sceneEl && this.sceneEl.getAttribute('debug');
+      var isDebugMode = this.sceneEl && this.sceneEl.hasAttribute('debug');
```

Comment:
Hm, but according to spec, if an attribute is considered boolean, the only way to disable it is to omit it, see https://html.spec.whatwg.org/#boolean-attributes:

> The values "true" and "false" are not allowed on boolean attributes. To represent a false value, the attribute has to be omitted altogether.

I also would consider this to be a boolean attribute and the "omit or true" mechanism makes sense IMHO as you'd normally leave it out and only specify it, if you want debug, no?